client/parser/test_helper.go

```diff
@@ -1,51 +1,83 @@
 package parser
 
 import (
+	"math"
 	"testing"
 
 	"alda.io/client/model"
 	"github.com/go-test/deep"
 )
 
-// A parseTestCase models the score updates that should result from parsing a
-// string of Alda code.
+// parseTestCase models a test of the `parser` module
+// parseTestCase also tests Score Updating which tests the `model` module
+// Perhaps this should live in another package that consumes both `parser` and
+// `model`, but for now we keep it here
 type parseTestCase struct {
-	label  string
-	given  string
-	expect []model.ScoreUpdate
+	label            string
+	given            string
+	expect           []model.ScoreUpdate // optional
```

Comment:
We could consider calling this `expectUpdates` for extra clarity

Reply:
Agreed. I actually made the change, then reverted it so that the PR would be more clean.

Now that you've done a first-pass, I'll do the renaming as it's own commit.